### DISCUSSION TITLE
UDM Pro IPTV with full IDS and IPS on

### DISCUSSION THREAD

**Discussion by Multied1** [fabianishere/udm-iptv#100] [General]
Hi Fabian,
I’m using the UDM pro with 1000/1000 fiber from KPN Netherlands. Before the UDM Pro I was using the Edgerouter 4 for 3 years in combination with IPTV successfully. 

Tomorrow I want to install your script on a dedicated subnet and direct cat 6 cable without any switches in between. From what I read this is adjustable via the wanted Br(x) settings.

Now I using the Expedia box V10 in front of the UDM Pro in DMZ mode and the IPTV signal is dedicated to the STB, so there is no multicast traffic on the UDM. 

Now my question: I have full IPS/IDS on, will this have any influence on the cpu and memory of the UDM Pro with IPTV installed?

Hope to hear from you and thanks in advance.
Br Ed

**Comment by reinierjh** [comment]
I have a UDM-SE and IPTV should not be a problem for "full IPS/IDS". I did not see any noticeable CPU usage from the IGMPPROXY application. (IPS/IDS is a real CPU hog but that's something different..)

**Comment by Multied1** [comment]
@reinierjh,
Great!!! Everything is up and running. At first the STB was in the wrong subnet and not the one I created specially for multicast traffic. Did the setup again and now it's working like a charm.

Thanks for the good work and have a nice weekend!
Br Ed
